.circleci/config.yml

```diff
@@ -57,6 +57,26 @@ jobs:
       - run:
           name: Publish @next
           command: npm publish --tag next --access public
+      - run:
+          name: Extract version from package
+          command: |
+            echo "export PACKAGE_VERSION=$(node -p "require('./package.json').version")" >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Login to Docker Hub
+          command: echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
+      - run: 
+          name: Build :$PACKAGE_VERSION
```

Comment:
Let's make it specific that we're building the Docker image here and update subsequent steps as well

Reply:
Renamed steps to `Build docker image :tag` and `Publish docker image :tag` to be more specific.